tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -94,6 +94,7 @@ private Security(SecuritySettings settings = null, InstrumentationGateway instru
                     if (_waf != null)
                     {
                         _instrumentationGateway.InstrumentationGatewayEvent += InstrumentationGatewayInstrumentationGatewayEvent;
+                        Log.Information("DDAS-0014-00: AppSec has started");
```

Comment:
Definitely Info? Do we always want to write here?

Reply:
Yes, doesn't really add much value, so removed in 4a501544a